src/webui/abstractwebapplication.cpp

```diff
@@ -258,14 +259,23 @@ void AbstractWebApplication::translateDocument(QString& data)
 
             QString translation = word;
             if (isTranslationNeeded) {
-                size_t context_index = 0;
-                while ((context_index < context_count) && (translation == word)) {
+                if (regex.cap(4).length() > 0) {
```

Comment:
Create variable with appropriate name for regex.cap(4) result to increase readability of next statements.